{
  "TargetAPK": {
    "Name": "python-3.11",
    "Version": "3.11.1-r5",
    "OriginPackageName": "python-3.11"
  },
  "Findings": [
    {
      "Package": {
        "ID": "2e7f7c7d2b673fdf",
        "Name": "python-3.11",
        "Version": "3.11.1-r5",
        "Type": "apk",
        "Location": "/.PKGINFO",
        "PURL": "pkg:apk/wolfi/python-3.11@3.11.1-r5?arch=x86_64\u0026origin=python-3.11"
      },
      "Vulnerability": {
        "ID": "CVE-2023-27043",
        "Severity": "Medium",
        "Aliases": [
          "CVE-2023-27043"
        ],
        "FixedVersion": "3.11.8-r1"
      }
    },
    {
      "Package": {
        "ID": "2e7f7c7d2b673fdf",
        "Name": "python-3.11",
        "Version": "3.11.1-r5",
        "Type": "apk",
        "Location": "/.PKGINFO",
        "PURL": "pkg:apk/wolfi/python-3.11@3.11.1-r5?arch=x86_64\u0026origin=python-3.11"
      },
      "Vulnerability": {
        "ID": "CVE-2023-40217",
        "Severity": "Medium",
        "Aliases": [],
        "FixedVersion": ""
      }
    },
    {
      "Package": {
        "ID": "2e7f7c7d2b673fdf",
        "Name": "python-3.11",
        "Version": "3.11.1-r5",
        "Type": "apk",
        "Location": "/.PKGINFO",
        "PURL": "pkg:apk/wolfi/python-3.11@3.11.1-r5?arch=x86_64\u0026origin=python-3.11"
      },
      "Vulnerability": {
        "ID": "CVE-2023-41105",
        "Severity": "High",
        "Aliases": [],
        "FixedVersion": ""
      }
    },
    {
      "Package": {
        "ID": "2e7f7c7d2b673fdf",
        "Name": "python-3.11",
        "Version": "3.11.1-r5",
        "Type": "apk",
        "Location": "/.PKGINFO",
        "PURL": "pkg:apk/wolfi/python-3.11@3.11.1-r5?arch=x86_64\u0026origin=python-3.11"
      },
      "Vulnerability": {
        "ID": "CVE-2023-6597",
        "Severity": "High",
        "Aliases": [
          "CVE-2023-6597"
        ],
        "FixedVersion": "3.11.8-r0"
      }
    },
    {
      "Package": {
        "ID": "2e7f7c7d2b673fdf",
        "Name": "python-3.11",
        "Version": "3.11.1-r5",
        "Type": "apk",
        "Location": "/.PKGINFO",
        "PURL": "pkg:apk/wolfi/python-3.11@3.11.1-r5?arch=x86_64\u0026origin=python-3.11"
      },
      "Vulnerability": {
        "ID": "CVE-2024-0450",
        "Severity": "Medium",
        "Aliases": [
          "CVE-2024-0450"
        ],
        "FixedVersion": "3.11.8-r0"
      }
    },
    {
      "Package": {
        "ID": "2e7f7c7d2b673fdf",
        "Name": "python-3.11",
        "Version": "3.11.1-r5",
        "Type": "apk",
        "Location": "/.PKGINFO",
        "PURL": "pkg:apk/wolfi/python-3.11@3.11.1-r5?arch=x86_64\u0026origin=python-3.11"
      },
      "Vulnerability": {
        "ID": "GHSA-5mwm-wccq-xqcp",
        "Severity": "Unknown",
        "Aliases": [],
        "FixedVersion": "3.11.8-r1"
      }
    },
    {
      "Package": {
        "ID": "2e7f7c7d2b673fdf",
        "Name": "python-3.11",
        "Version": "3.11.1-r5",
        "Type": "apk",
        "Location": "/.PKGINFO",
        "PURL": "pkg:apk/wolfi/python-3.11@3.11.1-r5?arch=x86_64\u0026origin=python-3.11"
      },
      "Vulnerability": {
        "ID": "GHSA-797f-63wg-8chv",
        "Severity": "Unknown",
        "Aliases": [],
        "FixedVersion": "3.11.8-r0"
      }
    },
    {
      "Package": {
        "ID": "2e7f7c7d2b673fdf",
        "Name": "python-3.11",
        "Version": "3.11.1-r5",
        "Type": "apk",
        "Location": "/.PKGINFO",
        "PURL": "pkg:apk/wolfi/python-3.11@3.11.1-r5?arch=x86_64\u0026origin=python-3.11"
      },
      "Vulnerability": {
        "ID": "GHSA-jm46-725r-hh9v",
        "Severity": "Unknown",
        "Aliases": [],
        "FixedVersion": "3.11.8-r0"
      }
    },
    {
      "Package": {
        "ID": "e65bf8c01f6ac635",
        "Name": "pip",
        "Version": "22.3.1",
        "Type": "python",
        "Location": "/usr/lib/python3.11/ensurepip/_bundled/pip-22.3.1-py3-none-any.whl",
        "PURL": "pkg:pypi/pip@22.3.1"
      },
      "Vulnerability": {
        "ID": "GHSA-mq26-g339-26xf",
        "Severity": "Medium",
        "Aliases": [
          "CVE-2023-5752"
        ],
        "FixedVersion": "23.3"
      }
    },
    {
      "Package": {
        "ID": "a145349ac56ab1b1",
        "Name": "setuptools",
        "Version": "65.5.0",
        "Type": "python",
        "Location": "/usr/lib/python3.11/ensurepip/_bundled/setuptools-65.5.0-py3-none-any.whl",
        "PURL": "pkg:pypi/setuptools@65.5.0"
      },
      "Vulnerability": {
        "ID": "GHSA-r9hx-vwmv-q579",
        "Severity": "High",
        "Aliases": [
          "CVE-2022-40897"
        ],
        "FixedVersion": "65.5.1"
      }
    }
  ],
  "GrypeDBStatus": {
    "built": "2024-06-11T01:29:53Z",
    "schemaVersion": 5,
    "location": "testdata/grypedb/5",
    "checksum": "sha256:d53b8f394bd155c9562f80d8b4128ff57b6e6fbfc122742454ad951363b9e05d",
    "error": null
  }
}
